pages/1.11/networking/index.md

```diff
@@ -59,17 +87,12 @@ and [Marathon-LB](/1.11/networking/marathon-lb/).
 | 0 hop load balancing               |     X     |         |             |
 | No single point of failure         |     X     |         |             |
 
-## Minuteman
-Minuteman is a distributed layer 4 virtual IP east-west load balancer that is installed by default.
-It's highly scalable and highly available, offering 0 hop load balancing, no single choke point,
-and tolerance to host failures.
 
 
-## Edge-LB
-Edge-LB builds upon on HAProxy. HAProxy provides base functionality such as load balancing for TCP and HTTP-based applications, SSL support, and health checking. In addition, Edge-LB provides first class support for zero downtime service deployment strategies, such as blue/green deployment. Edge-LB subscribes to Mesos and updates HAProxy configuration in real time.
+# A Note on Software Re-architecture
+In DC/OS 1.11 most of the networking components such `dcos-dns`, `dcos-l4lb`, `dcos-overlay`, are ErLang applications that run as part of a single ErLang VM called `dcos-net`. `dcos-net` is itself a systemD unit that runs on all nodes in the cluster. It is important to note that prior to DC/OS 1.11 each of the applications `dcos-dns`, `dcos-l4lb`, and `dcos-overlay` were running in their own ErLang VM, with their own repositories. Prior to DC/OS 1.11 role of `dcos-dns` was fulfilled by `spartan`, `dcos-l4lb` was fulfilled by `minuteman` and `dcos-overlay` was fulfilled by `navstar`. In DC/OS 1.11 we decided to aggregate all these different ErLang VMs into a single ErLang VM, primarily because this operational-pattern is more idiomatic to running ErLang applications. The main advantage of following this operational-patter is that it has lead to better efficiency in terms of resource utilization (lower CPU consumption and higher throughput), and has also made the networking services a lot more robust and reliable, not to mention that this approach has made the code a lot more maintainable. 
```

Comment:
I'm not exactly sure why you included this since it is totally transparent to the user. I don't think this paragraph should be in the primary documentation.  This could be greatly simplified and put into a release note.

You do have a typo in this "operational-patter" should be "operational pattern"